tests/handlers/test_q30_handler.py

```diff
@@ -9,43 +9,79 @@ class TestQ30Handler(HandlerTestBase):
 
     def setUp(self):
         key = "percent_q30"
-        qc_config = {'name': 'Q30Handler', 'error': 70, 'warning': 80}
+        qc_config = {"name": "Q30Handler", "error": 70, "warning": 80}
         value_1 = {"lane": 1, "read": 1, "percent_q30": 82}
-        value_2 = {"lane": 1, "read": 2, "percent_q30": 90}
-        value_3 = {"lane": 1, "read": 3, "percent_q30": 50, "is_index_read": True}
+        value_2 = {"lane": 1, "read": 2, "percent_q30": 50, "is_index_read": True} 
+        value_3 = {"lane": 1, "read": 3, "percent_q30": 50, "is_index_read": True} 
+        value_4 = {"lane": 1, "read": 4, "percent_q30": 60}
+        value_5 = {"lane": 2, "read": 1, "percent_q30": 90}
+        value_6 = {"lane": 2, "read": 2, "percent_q30": 50, "is_index_read": True}
+        value_7 = {"lane": 2, "read": 3, "percent_q30": 50, "is_index_read": False}
```

Comment:
```suggestion
        value_7 = {"lane": 2, "read": 3, "percent_q30": 50}
```
Since `is_index_read = False` is default there is no reason to write this out right?